solidity/contracts/Zoltar.sol

```diff
@@ -196,11 +194,12 @@ contract Zoltar {
 			ReputationToken(address(universe.reputationToken)).burn(migrator, amount);
 		}
 
-		uint192 childUniverseId = uint192((universeId << 2) + outcome + 1);
-		Universe memory childUniverse = universes[childUniverseId];
-		ReputationToken(address(childUniverse.reputationToken)).mint(recipient, amount);
+		for (uint8 i = 1; i < Constants.NUM_OUTCOMES + 1; i++) {
+			if (singleOutcome < 3 && i != singleOutcome + 1) continue;
+			uint192 childUniverseId = (universeId << 2) + i;
+			Universe memory childUniverse = universes[childUniverseId];
+			ReputationToken(address(childUniverse.reputationToken)).mint(recipient, amount);
```

Comment:
This has me wondering if REP should be an ERC1155 token instead of an ERC20, where the token ID is the universe ID.  This way there is a single "REP" token address throughout all forks rather than having a new one each time.

Reply:
Token ID would be universe + outcome I think. Would be easy to do since we have that pattern already established for share tokens. ERC1155 is potentially a UX downgrade for ethereum ecosystem stuff (trading etc) though as it never gained large scale adoption. On the other hand it does mean only one approval for Augur specific UX stuff. I'm ambivalent on it but open to this for sure if we have strong opinions on any level.